administrator/language/de-DE/com_actionlogs.ini

```
@@ -35,7 +35,7 @@ COM_ACTIONLOGS_IP_ADDRESS_DESC="IP-Adresse absteigend"
 COM_ACTIONLOGS_IP_INVALID="ungültige IP-Adresse"
 COM_ACTIONLOGS_IP_LOGGING_LABEL="IP-Protokollierung"
 COM_ACTIONLOGS_LOG_EXTENSIONS_LABEL="Ereignisprotokollierung"
-COM_ACTIONLOGS_LOG_VERBS_LABEL="Zu protokollierende API Verben"
+COM_ACTIONLOGS_LOG_VERBS_LABEL="Zu protokollierende Methoden"
```

```suggestion
COM_ACTIONLOGS_LOG_VERBS_LABEL="Zu protokollierende Request Methoden"
```
Basierend auf dem Original
> COM_ACTIONLOGS_LOG_VERBS_LABEL="Request Methods To Log"